test/include/seqan3/test/tmp_filename.hpp

```diff
@@ -116,22 +110,22 @@ class tmp_file_name
     explicit tmp_file_name(const char * f_name)
     {
         if (f_name == nullptr)
-            throw fs::filesystem_error("Empty file name!",
+            throw filesystem::filesystem_error("Empty file name!",
```

Comment:
Please align the wrapped lines.
I know we should not be burdened with this, but until #172 is in, we should to do it per hand if we can.